cves/CVE-2015-5144.yml

```diff
@@ -160,8 +162,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: Yes, any automated tool that tested for SQL injections could have discovered the vulerability.
```

Comment:
@moonj-kr Would a test for XSS also work?

Reply:
Yes I believe it should actually. Just added it, thanks!